core/camel-core-model/src/main/java/org/apache/camel/model/NoOutputDefinition.java

```diff
@@ -29,4 +29,10 @@ public List<ProcessorDefinition<?>> getOutputs() {
         return Collections.emptyList();
     }
 
+    public NoOutputDefinition() {
+    }
+
+    NoOutputDefinition(NoOutputDefinition source) {
```

Comment:
Should all the copy constructors be  `protected` rather than package private ? This won't change much as afaik all the models are in the same package, but the idea of this constructor is that it will be called by children classes, so protected makes more sense here.